lib/Test/Nginx/Socket.pm

```diff
@@ -1754,7 +1756,37 @@ sub parse_response($$$) {
 
         my $decoded = '';
         while (1) {
-            if ( $raw =~ /\G 0 [\ \t]* \r\n \r\n /gcsx ) {
+            if (@trailers == 0 && $raw =~ /\G 0 [\ \t]* \r\n \r\n /gcsx ) {
+                if ( $raw =~ /\G (.+) /gcsx ) {
+                    $left = $1;
+                }
+
+                last;
+
+            } elsif (@trailers > 0 && $raw =~ /\G 0 [\ \t]* \r\n /gcsx) {
+                # skip HTTP Trailer
```

Comment:
maybe it's better to set the trailers to res header?

Reply:
It is supported. See `my @trailers = $res->header('Trailer');` above.